test/hotspot/jtreg/runtime/Metaspace/elastic/TestMetaspaceAllocation.java

```diff
@@ -45,19 +45,21 @@
  * @run main/othervm -Xbootclasspath/a:. -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI TestMetaspaceAllocation
  */
 
+import jdk.test.lib.Unit;
+
 public class TestMetaspaceAllocation {
 
     public static void main(String[] args) {
 
         MetaspaceTestContext context = new MetaspaceTestContext();
-        MetaspaceTestArena arena1 = context.createArena(false, 1024 * 1024 * 4);
-        MetaspaceTestArena arena2 = context.createArena(true,1024 * 1024 * 4);
+        MetaspaceTestArena arena1 = context.createArena(false, 32L * Unit.valueOf("M").size());
+        MetaspaceTestArena arena2 = context.createArena(true, 32L * Unit.valueOf("M").size());
```

Comment:
Why not just `Unit.M.size()` ?

Reply:
Good point - direct access is less error prone anyway. I can update it.